src/mono/wasm/build/WasmApp.InTree.targets

```diff
@@ -1,6 +1,7 @@
 <Project>
   <!-- This depends on the root Directory.Build.targets imported this file -->
   <UsingTask TaskName="MonoAOTCompiler" AssemblyFile="$(MonoAOTCompilerTasksAssemblyPath)" />
+  <UsingTask TaskName="ILStrip" AssemblyFile="$(MonoTargetsTasksAssemblyPath)" />
```

Comment:
You would need this in WasmApp.LocalBuild.targets too.

Reply:
It has already been specified there. See https://github.com/dotnet/runtime/blob/main/src/mono/wasm/build/WasmApp.LocalBuild.targets#L67